### PR TITLE
fix: remove starknet foundry folder

### LINKED ISSUE #298: Get rid of starknet-foundry folder
`crates` folder could be in the top directory

### PR DESCRIPTION
**Pull request by julio4**
Closes #298

## Introduced changes

- Move all content of folder `/starknet-foundry` to `/`
- Fix all paths in pipeline, documentation, scripts and includes.

## Breaking changes

- `/starknet-foundry` -> `/`

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,8 +24,8 @@ jobs:
       - uses: software-mansion/setup-scarb@v1.0.0
         with:
           scarb-version: ${{ env.SCARB_VERSION }}
-      - run: cargo test --lib --manifest-path ./starknet-foundry/Cargo.toml -p forge
-      - run: cargo test integration --manifest-path ./starknet-foundry/Cargo.toml -p forge
+      - run: cargo test --lib --manifest-path ./Cargo.toml -p forge
+      - run: cargo test integration --manifest-path ./Cargo.toml -p forge
```
Comment (cptartur) [inline .github/workflows/ci.yml:28]: Not sure, but we may not need to pass `--manifest-path` now if the `Cargo.toml` is in the top-level anyways

Comment (cptartur) [reply .github/workflows/ci.yml:28]: Same goes for code below, though it requires veryfing

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,5 @@
 pyproject.toml
 .DS_Store
 .tool-versions
+
```
Comment (cptartur) [inline .gitignore:5]: Nit pick, but we don't need newline here

**Review by cptartur — COMMENTED** [review]
Thanks for contributing 👍 

You may want to merge/rebase your changes since there are conflicts with the directories added since the PR creation.

File: .gitignore (hunk 2 above)
Comment (Radinyn) [reply .gitignore:5]: I need a newline here

**Comment by julio4** [comment]
Done!

**Review by Radinyn — APPROVED** [review]
LGTM🚀🚀

**Review by war-in — APPROVED** [review]

**Comment by MaksymilianDemitraszek** [comment]
❤️ Thank you for contributing!

**Pull request merged**